### DISCUSSION TITLE
Welcome to danish-foundation-models Discussions!

### DISCUSSION THREAD

**Discussion by KennethEnevoldsen** [centre-for-humanities-computing/danish-foundation-models#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by de-Blanck** [comment]
Hello and awesome initiave!! 

How can I in any way, help or support the initiative voluntarily and unpaid? 

Thank you 
Rune 

(P.S. danish-native and based in DK).

**Reply by KennethEnevoldsen** [reply]
Hej Rune!

Ingen får betaling igennem projektet, men alle som contributor pt. får løn igennem de institutioner de arbejder ved. Håber det svarer på spørgsmåls eller spørg endelig mere ind.

**Comment by Hammad611** [comment]
Hello
How can i get information about any PhD vaccancy as the reaearch work of NLP in DFM is nmuch alligned with my research backgroud. I will share the details of my work when guided

Best

**Reply by KennethEnevoldsen** [reply]
Thanks for reaching out. For that, I would suggest reaching out to one of the PIs:

- Kristoffer Laigaard Nielbo ([kln@cas.au.dk](mailto:kln@cas.au.dk))
- Peter Schneider-Kamp ([petersk@imada.sdu.dk](mailto:petersk@imada.sdu.dk))
- Desmond Elliott ([de@di.ku.dk](mailto:de@di.ku.dk))
- Bolette Sandford Pedersen ([bspedersen@hum.ku.dk](https://www.foundationmodels.dk/(mailto:bspedersen@hum.ku.dk))
